test/Merging/indirect_value.c

```diff
@@ -0,0 +1,33 @@
+// RUN: %llvmgcc -emit-llvm -g -c -o %t.bc %s
+// RUN: rm -rf %t.klee-out
+// RUN: klee --output-dir=%t.klee-out --use-merge --debug-log-merge --search=nurs:covnew --use-batching-search %t.bc 2>&1 | FileCheck %s
+
+// CHECK: generated tests = 2
+#include <stdlib.h>
+#include <stdio.h>
+
+#include <klee/klee.h>
+
+int main(int argc, char** argv) {
+  (void)argc; (void)argv;
```

Comment:
Can you remove this line?  You don't have it in other test files and I don't think it triggers a warning without it, does it?